txnsync/txngroups.go

```diff
@@ -25,7 +27,10 @@ import (
 	"github.com/algorand/go-algorand/protocol"
 )
 
-func encodeTransactionGroups(inTxnGroups []transactions.SignedTxGroup) ([]byte, error) {
+const compressionSpeed = 23071093.0 // bytes per second
```

Comment:
`compressionSpeed ` -> `estimatedGzipCompressionSpeed`
`compressionSavings` -> `estimatedGzipCompressionGains`
in addition to the above, also add proper description for each of them, as well as the name of the benchmark that is used to calculate them